### PR TITLE
Add devise for user

### PR DESCRIPTION
**Pull request by ritamma**
Add devise gem for simple registration, create model and table for users, create static pages controller for root route.

### CODE REVIEW

File: test/controllers/static_pages_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
```
Comment (fooooxmr) [inline test/controllers/static_pages_controller_test.rb]: Remove this test. We need to use Rspec.

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (fooooxmr) [inline test/models/user_test.rb]: Remove this test. We need to use Rspec.

File: test/fixtures/users.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```
Comment (fooooxmr) [inline test/fixtures/users.yml]: This fixture also do not needed.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: test/controllers/static_pages_controller_test.rb (hunk 1 above)
Comment (ritamma) [reply test/controllers/static_pages_controller_test.rb]: Done

File: test/fixtures/users.yml (hunk 3 above)
Comment (ritamma) [reply test/fixtures/users.yml]: Done

File: test/models/user_test.rb (hunk 2 above)
Comment (ritamma) [reply test/models/user_test.rb]: Done

**Review by fooooxmr — APPROVED** [review]

**Review by MariaGrechko — APPROVED** [review]

**Review by ShalaevPavel — APPROVED** [review]

**Review by Ikl0 — APPROVED** [review]

**Pull request merged**